static/app/components/nav/config.tsx

```diff
@@ -209,6 +212,34 @@ export function createNavConfig({organization}: {organization: Organization}): N
       {label: t('Alerts'), to: `/${prefix}/alerts/rules/`, icon: <IconSiren />},
     ],
     footer: [
+      {
+        label: t('Help'),
+        icon: <IconQuestion />,
+        menu: [
```

Comment:
This feature might get tricky for some of the menus that sometimes use getsnetry hooks, but we can cross that bridge later 😅